docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -75,7 +75,7 @@ The following table lists the properties that can be configured:
 | port | Integer | the SMTP server port | `25`
 | username | String | the username |
 | password | String | the password |
-| ssl | Boolean | Enables or disables SSL | `false`
+| ssl | Boolean | Enables or disables SSL. When enabled, you must add `quarkus.ssl.native=true` config option in `application.properties` when building a native image. | `false`
```

Comment:
I would just say: "If enabled, you must also set `quarkus.ssl.native` to true when building a native image.".

You are already in the config context so I don't think we need more precision. I think I would also add a link to the SSL + native image guide (`More information`).

Reply:
A link is definitely needed. I’ll add it and push. Thanks.

Reply:
Maybe you can just add the link on `set quarkus.ssl.native to true`. It will be shorter.